kvrocks.conf

```diff
@@ -576,7 +576,7 @@ rocksdb.target_file_size_base 128
 # The default and the minimum number is 2, so that when 1 write buffer
 # is being flushed to storage, new writes can continue to the other
 # write buffer.
-# If max_write_buffer_number > 3, writing will be slowed down to
+# If max_write_buffer_number < 3, writing will be slowed down to
```

Comment:
This is greater than 3, please refer to:
https://github.com/facebook/rocksdb/blob/36f48d16a8a14fb3bb22c6d9ed5ba4ad1f01bc50/include/rocksdb/advanced_options.h#L282

Reply:
Hmm, you are right, thanks! It's fully unlogical for me, but maybe this are ok )